Added Cauldron outlet, modified Ionocraft backpack 添加了炼药锅口，修改了飘升机背包

WhereisFff opened the pull request:
- Resolved #2953

WhereisFff committed:
又改了改飘升机背包

WhereisFff committed:
给可过滤的格子添加了物品上限和相应的功能，减弱了动力滑轨的吸力接近滑轨站，让飘升机背包根据电网中的剩余功率决定充电功率和速率并修改了配方，添加了前面几个更改的tooltip

WhereisFff committed:
补多玩家，修格式

WhereisFff committed:
添加了实体：炼药锅口(CauldronMouth)，并实现了功能，但还没有模型

WhereisFff committed:
重命名了炼药锅口(CauldronOutlet)

Gu-ZT commented on src/main/java/dev/dubhe/anvilcraft/init/entity/ModEntities.java:
```diff
@@ -117,6 +119,12 @@ public class ModEntities {
         .register();
 
+    public static final EntityEntry<? extends CauldronOutletEntity> CAULDRON_MOUTH = AnvilCraft.REGISTRATE
+        .<CauldronOutletEntity>entity("cauldron_mouth", CauldronOutletEntity::new, MobCategory.MISC)
```
此处 “cauldron_mouth” 应修改为 “cauldron_outlet”

Gu-ZT committed:
fix(entity):修复CauldronOutlet实体注册问题

- 将 CAULDRON_MOUTH 实体注册更正为 CAULDRON_OUTLET
- 更新实体构造函数中的注册引用
- 确保实体渲染器正确关联到新注册名
- 修正实体类型引用以匹配新的注册标识符

Gu-ZT committed:
refactor(cauldron): 优化炼药锅检测逻辑并清理代码

- 重新格式化炼药锅方块状态检查条件
- 移除多余的空行以提高代码可读性
- 保持原有功能不变，提升代码整洁度

Gu-ZT committed:
fix(cauldron): 添加 Nullable 注解以明确返回值可能为空

Gu-ZT committed:
feat(lang): 添加新实体和物品本地化键值

- 在 en_ud.json 和 en_us.json 中添加了 cauldron_outlet 实体的本地化
- 更新了 ionocraft_backpack 的工具提示文本
- 移除了过时的电容器容量描述
- 调整了过滤器界面的相关文本
- 修改了 ionocraft_backpack 合成配方中使用的材料标签从钛板为锡板
- 更新了相关进度条件以匹配新的合成配方要求
